### ISSUE TITLE
cursor not displaying on clipped screen

### ISSUE THREAD

**Issue by Pshemas** [LibVNC/x11vnc#92]
I've noticed cursor is not showing up on the clients if I "cast" a clipped display. Is there an option that would fix it?
I'm starting x11vnc with this:

x11vnc -noxdamage -viewonly -alwaysshared -shared -forever -clip 1920x1200+1600+0

Clipping is important for me as I have two displays and I want to "cast" only one of those to the clients.

**Comment by bk138** [comment]
Hi! Have you tried other options - do they make a difference?

**Comment by Pshemas** [comment]
which options can I try?

**Comment by bk138** [comment]
They're listed in the man page.

**Comment by Pshemas** [comment]
yeah, but there are like gazillion ;) options and I'm no expert - so any suggestions which options could be worthwhile would be welcome.

**Comment by Pshemas** [comment]
also - where's the man page? I've been browsing this one:
http://www.karlrunge.com/x11vnc/

**Comment by beanaroo** [comment]
`nocursorshape` seems to work for me. Though I'm having the problem of the pointer running along the edge of the client when the cursor is outside of the clipped area. Not sure if related.

**Comment by tusqasi** [comment]
`nocursorshape` does work for me.

**Comment by whinee** [comment]
this should be higher up in the search engine. sorry for digging up the past, but here's what im using

```sh
x11vnc -display :0 -clip 1440x900+0+90 -rfbport 5900 -rfbauth ~/.vnc/passwd -nocursorshape -forever
```

i used to use the following:

```sh
x11vnc -display :0 -clip 1440x900+0+90 -rfbport 5900 -rfbauth ~/.vnc/passwd -multiptr -forever
```

but that caused annoying duplicate pointers in my screen, which is really annoying.

again, thank you!!
